<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClustParaAdjustingDlg</class>
 <widget class="QDialog" name="ClustParaAdjustingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="lblWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="distWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="distLbl">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>Cutoff Height:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="distVal">
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="minPtsLbl">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>Num Clusters:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="minPtsVal">
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="sldrWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="distSldrWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QSlider" name="distSldr">
             <property name="singleStep">
              <number>5</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="minPtsSldrWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QSlider" name="minPtsSldr">
             <property name="singleStep">
              <number>5</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
